src/test/java/uk/gov/hmcts/reform/sscs/ccd/domain/BenefitTest.java

```diff
@@ -54,18 +54,7 @@ public void givenAnInvalidBenefitString_thenReturnTrue() {
             "ESA, Employment and Support Allowance (ESA)",
             "UC, Universal Credit (UC)",
             "JSA, Jobseeker’s Allowance (JSA)",
-            "DLA, Disability Living Allowance (DLA)",
-            "Carer's Allowance, Carer's Allowance",
-            "Attendance Allowance, Attendance Allowance",
-            "Bereavement Benefit, Bereavement Benefit",
-            "Industrial Injuries Disablement Benefit, Industrial Injuries Disablement Benefit",
-            "Maternity Allowance, Maternity Allowance",
-            "Social Fund, Social Fund",
-            "Income Support, Income Support",
-            "Bereavement Support Payment Scheme, Bereavement Support Payment Scheme",
-            "Industrial Death Benefit, Industrial Death Benefit",
-            "Pension Credit, Pension Credit",
-            "Retirement Pension, Retirement Pension"
```

Comment:
Did you mean to remove these tests?

Reply:
Yes, they are duplicated in the test above (or below)